content/docs/practices/histograms.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Histograms and summaries
+sort_rank: 4
+---
+
+# Histograms and summaries
+
+Histograms and summaries are more complex metric types. Not only
+creates a single histogram or summary a multitude of time series, it
```

Comment:
Well, also the subject is in the wrong position, and the verb is not conjugated correctly? I would just copy my suggestion :)